src/openrct2/localisation/StringIds.h

```diff
@@ -3911,8 +3911,10 @@ enum
 
     STR_COPY_INPUT_TO_CLIPBOARD = 6363,
 
+    STR_WINE_NOT_RECOMMENDED = 6368,
+
     // Have to include resource strings (from scenarios and objects) for the time being now that language is partially working
-    STR_COUNT = 32768
+    STR_COUNT = 32769
```

Comment:
I understand why you incremented this, but it shouldn't be incremented when you add strings.

Reply:
I see people make this mistake all the time, perhaps it would be a good idea to change this name to something like MAX_STR_COUNT instead?

Reply:
It's a good idea indeed, or to add a comment explaining. I'll see if I can tend to it today